tasks/eth/006-MCP-L1/SignFromJson.s.sol

```diff
@@ -0,0 +1,346 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "../../../script/SignFromJson.s.sol";
+import {SystemConfig} from "@eth-optimism-bedrock/src/L1/SystemConfig.sol";
+import {Constants, ResourceMetering} from "@eth-optimism-bedrock/src/libraries/Constants.sol";
+import {L1StandardBridge} from "@eth-optimism-bedrock/src/L1/L1StandardBridge.sol";
+import {L2OutputOracle} from "@eth-optimism-bedrock/src/L1/L2OutputOracle.sol";
+import {ProtocolVersion, ProtocolVersions} from "@eth-optimism-bedrock/src/L1/ProtocolVersions.sol";
+import {SuperchainConfig} from "@eth-optimism-bedrock/src/L1/SuperchainConfig.sol";
+import {OptimismPortal} from "@eth-optimism-bedrock/src/L1/OptimismPortal.sol";
+import {L1CrossDomainMessenger} from "@eth-optimism-bedrock/src/L1/L1CrossDomainMessenger.sol";
+import {OptimismMintableERC20Factory} from "@eth-optimism-bedrock/src/universal/OptimismMintableERC20Factory.sol";
+import {L1ERC721Bridge} from "@eth-optimism-bedrock/src/L1/L1ERC721Bridge.sol";
+import {Predeploys} from "@eth-optimism-bedrock/src/libraries/Predeploys.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+
+contract SignFromJson is OriginalSignFromJson {
+    address constant finalSystemOwner = 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A;
```

Comment:
Could we add comments above each of these with a link to its source of truth?

Reply:
This would be a permalink to the mainnet deploy config. For the guardian, final system owner, batch inbox, batch sender, proposer and challenger it could be good to have a link to etherscan

The idea is that we want to reduce the cognitive overhead of a reviewer so the chances of a bad code review happening are small

Reply:
Agreed, I'd suggest preferring links to the superchain registry when applicable, and falling back to the deploy config or etherscan for data not in the registry

Reply:
for links in the monorepo, I'm using the MCP commit as the version for the files